src/gax-internal/src/api_header.rs

```diff
@@ -48,7 +48,25 @@ impl XGoogApiClient {
         let gax_version = build_info::PKG_VERSION;
 
         format!(
-            "gl-rust/{rustc_version} gax/{gax_version} {}/{}",
+            "gl-rust/{rustc_version} gax/{gax_version} rest/{gax_version}-reqwest {}/{}",
```

Comment:
Q: why duplicate `{gax_version}`?

Ideally would we capture the version of `reqwest`?

Reply:
The spec (go/actools-regapic-tracking-metrics) says we can use either the gax version or the version of the http client.  And I cannot find a way to capture the version of `reqwest` or any other dependency.